core/iwasm/libraries/wasi-nn/wasi_nn_tensorflow.cpp

```diff
@@ -38,8 +38,43 @@ _load(graph_builder_array builder, graph_encoding encoding)
 
     if (interpreter == nullptr) {
         printf("failure: null interpreter \n");
-        return invalid_argument;
+        return missing_memory;
     }
 
+    return success;
+}
+
+uint32_t
+_set_input(tensor input_tensor)
+{
+    auto *input = interpreter->typed_input_tensor<float>(0);
+
+    if (input == nullptr) {
+        return missing_memory;
+    }
+
+    for (int i = 0; i < input_tensor.dimensions[0]; i++) {
+        input[i] = (float)input_tensor.data[i];
+    }
+
+    return success;
+    
+}
+
+
+uint32_t _compute( graph_execution_context context ){
+
+	interpreter->Invoke();
+
+    return success;
+}
+
+
+uint32_t _get_output(graph_execution_context context, uint32_t index, tensor tensor){
+
+    auto* output = interpreter->typed_output_tensor<float>(0);
```

Comment:
where is this output going?

Reply:
Made the change, we pass a pointer and modify it's value